dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json

```diff
@@ -26,5 +26,9 @@
   "DropTableCommand": "ddl",
   "TruncateTableCommand": "ddl",
   "AlterTableRecoverPartitionsCommand": "ddl",
-  "SetCatalogAndNamespace": "ddl"
+  "SetCatalogAndNamespace": "ddl",
```

Comment:
this seems not a ddl?

Reply:
the "SetCatalogAndNamespace" is the grammer of USE's simpleName.
In spark_version_3.1.2, this grammar is belong to Data Definition Statements.

Reply:
got it